gorapass/views.py

```diff
@@ -12,6 +12,10 @@
 def index(request):
     return HttpResponse('Hello, World. This is Naya and Brandi\'s super cool app.')
 
+def stamps(request):
+    stamp_model = list(Stamps.objects.values())
```

Comment:
This looks good. The only thing that comes to mind is what happens if the stamps list is empty. With this code, we will be given an empty list, where we might want to return a 404 instead. Not likely to happen, but I think it's a good habit. `django.shortcuts` provides an equivalent `get_list_or_404` that does the same as `get_object_or_404`, but for collections.

What do you think?

Reply:
Thinking on this some more, I think that returning an empty list makes sense. We 404 when we can't find something specific. We can find the hikes, there just aren't any. I think this is good.

Reply:
I agree. I think we should return an empty list. We are aligned 👍